src/current/cockroachcloud/folders.md

```diff
@@ -346,21 +214,13 @@ Your service account must have permission to move folders at both the source and
 When you move a folder, users or service accounts who had access to the previous location through inheritance may lose access to its descendant folders and clusters. Roles granted directly on a folder or a cluster do not change when the folder or cluster is moved.
 {{site.data.alerts.end}}
 
-1. To move a folder and its contents into another folder:
+To move a folder and its contents into another folder:
 
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    curl --request PATCH \
-           --url 'https://cockroachlabs.cloud/api/v1/folders/<folder-id>' \
-           --header 'Authorization: Bearer <api-key>' \
-           --header 'content-type: application/json' \
-           --data '{"parent_id":"<parent_id>"}'
-    ~~~
-
-    Replace:
-    - `<folder-id>`: The ID of the folder.
-    - `<api-key>`: Your service account's API key.
-    - `<parent-id>`: The ID of the new parent folder.
+1. Go to the **Clusters** page. The folders and clusters at the root of the organization are shown.
+1. Browse to the folder that contains the folder you want to move.
```

Comment:
Will this suffice? If you want to move an root-level folder, it seems the folder won't be in another folder, unless I'm not understanding.

```suggestion
1. Browse to the location of the folder that you want to move.
```